### ISSUE TITLE
Total Update version Blender3D

### ISSUE THREAD

**Issue by ZiCVIZ** [overmindstudios/BlenderUpdater#42]
Hi
Can you update different blender versions in different folders of one button ?
so that you can configure different folders once
It would also be nice to do updates for normal non-beta versions

PS:
Thanks for your work !

**Comment by XeonG** [comment]
yeah its a useful tool... would be good to see it improved with more options to do things like that

**Comment by tobkum** [comment]
I'll look into that as soon as I have more time on my hands :) Great idea!
